android/build.gradle

```diff
@@ -97,7 +99,7 @@ afterEvaluate { project ->
     }
 
     task androidSourcesJar(type: Jar) {
-        classifier = 'sources'
+        archiveClassifier = 'sources'
```

Comment:
i have seen that `classifier` is deprecated so it might be nice if you changed it also in the `androidJavadocJar` task .
PS: i think that `configureReactNativePom` method is not used anymore

Reply:
@adamsatouri updated